docs/guide/cleanup.md

```diff
@@ -131,3 +131,23 @@ The fix here is often that we either:
 - Move to a dependency which is leaner and is more focused.
 
 In this situation, it is worth looking for a lighter (and hopefully faster) alternative.
+
+### Finding Other Dependents of Potential Packages
+
+Once you identify a package that has potential for replacement, create an issue in our [cleanup-repository](https://github.com/es-tooling/ecosystem-cleanup) to inform other community members.
+
+Next, use[fuzzymas tool](https://github.com/fuzzyma/e18e-tools) to find the dependents of the package. You can find documentation on how to use the tool in its README.
+
+Here’s an example that displays the top 100 dependents of `lodash`:
+
+```bash
+npx github:Fuzzyma/e18e-tools lodash -n 100 -U https://npm.devminer.xyz/registry
```

Comment:
Should we point out something like who to contact or what to do when the CouchDB URL is not reachable?

I was thinking of providing some docs on how to host the database itself (in which someone would have to privately request the data to someone), but I don't think that's relevant enough for now.

Reply:
I think if they really want to do that, they can join the discord and just ask :)